migrate/20231207_storage_device.rb

```diff
@@ -7,6 +7,7 @@
       column :name, :text, null: false
       column :total_storage_gib, :Integer, null: false
       column :available_storage_gib, :Integer, null: false
+      column :enabled, :bool, null: false, default: true
```

Comment:
Patch to a migration is hazardous, since if it applied in its previous version, it will not have its changed version applied. A mistake?